includes/polling/ports.inc.php

```diff
@@ -744,10 +744,14 @@
         $fields['ifOutErrors_rate'] = $port['ifOutErrors_rate'];
         $fields['ifInOctets_rate'] = $port['ifInOctets_rate'];
         $fields['ifOutOctets_rate'] = $port['ifOutOctets_rate'];
+        $fields['ifInBits'] = $port['stats']['ifInBits_rate'];
```

Comment:
Why are these needed?  If you want bits, you can just take ifOutOctets_rate / 8 when graphing the data.